element-connector/src/main/java/org/eclipse/californium/elements/tcp/TlsServerConnector.java

```diff
@@ -58,6 +59,7 @@ public TlsServerConnector(KeyManagerFactory keyManagerFactory, InetSocketAddress
 
 	@Override protected void onNewChannelCreated(Channel ch) {
 		SSLEngine sslEngine = sslContext.createSSLEngine();
+		sslEngine.setWantClientAuth(true);
```

Comment:
For `DTLSConnector` this is configurable. I do not know whether you have given configuration of the TLS connector much thought yet ...